src/realm-settings/keys/KeysListTab.tsx

```diff
@@ -32,18 +35,17 @@ type KeysListTabProps = {
 };
 
 export const KeysListTab = ({ realmComponents }: KeysListTabProps) => {
-  const { t } = useTranslation("roles");
+  const { t } = useTranslation("realm-settings");
   const history = useHistory();
-  const { url } = useRouteMatch();
 
   const [key, setKey] = useState(0);
   const [publicKey, setPublicKey] = useState("");
   const [certificate, setCertificate] = useState("");
   const [filterDropdownOpen, setFilterDropdownOpen] = useState(false);
-  const [filterType, setFilterType] = useState("Active keys");
+  const [filterType, setFilterType] = useState<string>(FILTER_OPTIONS[0]);
```

Comment:
I think you can omit the type here, as it should be inferrable by the default you are providing.
```suggestion
  const [filterType, setFilterType] = useState(FILTER_OPTIONS[0]);
```